src/services/Logger.ts

```diff
@@ -0,0 +1,25 @@
+export class Logger {
+    prefix: string;
+
+    constructor(prefix: string) {
+        this.prefix = prefix;
+    }
+    
+    /**
+     * Logs message with non-error coloring. Also includes timestamp.
+     *
+     * @param info - message to log
+     */
+    logInfo(info: string): void {
+        console.log('\x1b[33m%s\x1b[0m', `[${new Date().toLocaleString()}] - ${this.prefix}: ${info}`);
```

Comment:
What is `'\x1b[33m%s\x1b[0m'`. Is it worth naming as a constant, so that the reader knows that that does? 


Also maybe for future, we should decide whether we wanna log the timestamp that the client sent the message, or the timestamp that our backend got the message (if we ever even add a database i guess haha)

Reply:
Maybe I can just put documentation for it in the comments for the function since it's just being used here?

Reply:
Isnt it being used in line 27 too?

Reply:
It's a different one. I added a comment for both of them.